sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+
+commit https://github.com/intel/llvm/commit/38e663ecd37de513d8e31afdfdf245cf8c9d17f0
+    [SYCL] Declare __devicelib_assert_read only when fallback assert is enabled (#13241)
+    Is there any particular user-visible bug associated with this?
+
+commit https://github.com/intel/llvm/commit/4b993a7b32f7743980bce646765a1b427b0996b6
+    Revert "[SYCL][Driver] Link with sycl libs at link step of clang-cl -fsycl (#12793)" (#13326)
+    revert commit https://github.com/intel/llvm/commit/seems to be a part of a previous release
+
+commit https://github.com/intel/llvm/commit/4b14d706d93891cdb5b0e6a8d4b0b027c1d54ab8
+    [SYCL][DeviceSanitizer] Use -asan-constructor-kind=none to disable ctor/dtor (#13259)
+    was the bug really user-visible?
+commit https://github.com/intel/llvm/commit/2442ef047a4e9e9c135beed18a92029e1aad6cad
+    [DeviceSanitizer] Disable handling no return calls (#14652)
+    // bugfix?
+
++UR commit below
+
+## New Features
+
+### SYCL Compiler
+
+- Added `-fsycl-range-rounding` command line option which allows to
+  control range rounding feature. In comparison with previously available
+  `-fsycl-disable-range-rounding` command line option and
+  `__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__` macro the new flag also allows
+  to _force_ range rounding which will complete disable generation of
+  non-rounded kernels, thus improving binary size. intel/llvm#12715
+- Added `-fsycl-exp-range-rounding` command line option that enables
+  experimental range rounding mode in which range rounding is performed across
+  all dimensions. intel/llvm#12690
+- Added support for the so-called [new offloading model](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/design/OffloadDesign.md).
+  It can be enabled by `--offload-new-driver` command line option and provides
+  a better infrastructure for us. In the future we expect to leverage that
+  infrastructure to improve link times by reducing amount of I/O used by the
+  compiler and amount of external processes that it spawns.
+  Foundation for this work has been performed in previous release time frame and
+  the following list of PRs only includes those done within this release
+  time frame. intel/llvm#14252 intel/llvm#13394 intel/llvm#13648 intel/llvm#13687
+  intel/llvm#14001 intel/llvm#14006 intel/llvm#14101 intel/llvm#14151
+  intel/llvm#14253 intel/llvm#14177 intel/llvm#13672 intel/llvm#13688
+  intel/llvm#13579 intel/llvm#13869 intel/llvm#14102 intel/llvm#14541
+  intel/llvm#14541 intel/llvm#13898 intel/llvm#14143
+  allow us to improve link time by reducing amount of external processes and
+  temporary files used by the compiler. **Do we need to list PRs here?** There
+  were many of them and some of them were merged in scope of a previous release.
+- Added `-fsycl-fp64-conv-emu` command line option which allows to enable
+  partial (only conversion operations are supported) emulation of `double` data
```

Comment:
```suggestion
- Added `-fsycl-fp64-conv-emu` command line option which allows the enabling
  of partial (only conversion operations are supported) emulation of `double` data
```

Reply:
Applied in 1fe2c554292d3a9d90f952c02ff4b8e3dfaab706